tests/ops/functions/test_equal.py

```diff
@@ -1568,3 +1568,99 @@ def test_sum_with_multi_wire_operations(self, base_list1, base_list2, res):
         op1 = qml.sum(*base_list1)
         op2 = qml.sum(*base_list2)
         assert qml.equal(op1, op2) == res
+
+
+f1 = qml.pulse.pwc([0.3, 1.0])
+
+
+def f2(p, t):
+    return p[0] * t**2 + p[1]
+
+
+class TestParametrizedEvolutionComparisons:
```

Comment:
Could you add docstrings to the class and functions? :)

Reply:
Added :)

Reply:
Thanks :100: